### ISSUE TITLE
Component & Serials

### ISSUE THREAD

**Issue by teamuppolska** [grokability/snipe-it#9015]
When I add a component, I can add the serial number, but also the quantity of the components. However, the serial number remains one. There are no components with the same serial number. I should be able to add e.g. 3 motherboards with different serial numbers, but if I later want to check how many motherboards I have, I should see them on one place. Now I have to add separate components and I don't know how many of the same component I have.

Snipe-it 5.0.12

**Comment by jonathanspw** [comment]
Join the club in #4421...

**Comment by Neanjah** [comment]
#4421 has already been closed but I think being able to add multiple serial numbers would still be a great feature.

This is my workaround for this issue:
![image](https://user-images.githubusercontent.com/95951961/145973916-579a9626-71e9-4bb7-8a44-b4cada5ca534.png)

I was able to fit 10 serial numbers with 14 characters in the text box. Then I just take note of the exact serial number of the component in the Maintenance entry.

I wish they'd automatically checkout a component when a Maintenance entry is added.

**Comment by snipe** [comment]
If they need serial numbers, just make them assets.

**Comment by Neanjah** [comment]
@snipe wouldn't that defeat the purpose of having a separate Components section?

**Comment by Prodeguerriero** [comment]
@snipe Quick question, because I cannot find the answer anywhere else: I am adding all my spare parts as assets, as you have pointed out more than once, because I want to track the serial numbers.

What I am failing to figure out is a use case where adding a component to the database makes sense. In other words, how do you suggest to use the "component" type? I just want to make sure I understand how you intend the tool to be used, so that I can apply the same logic (if possible).

Thanks for everything!

**Comment by kuzi-moto** [comment]
I had similar questions as Prodeguerriro. I have looked into the documentation and wasn't able to find anything regarding what a component is and what it should be used for. Seems like a cross between consumables and assets, but it is somewhat unclear as to the proper use.

It's not a big deal for me, and it's not going to stop me from using Snipe-IT because the work-around is a fine solution. Mostly just curious so I know how to use it most effectively.

**Comment by snipe** [comment]
Proper use would be, for example, sticks of RAM. Many people don't track the serials on them, and they're too small to put a label on.

**Comment by Prodeguerriero** [comment]
I see, it does make sense doing it like that in a bigger operation, where the volume justifies being a bit more loose with tracking. All clear, thanks!

**Comment by Prodeguerriero** [comment]
@snipe 

> Proper use would be, for example, sticks of RAM. Many people don't track the serials on them, and they're too small to put a label on.

I have now tinkered with this a bit more, and I remembered where my original confusion came from. If the assumption is that one uses `Components` for things that do not need an asset tag and are not tracked via serial number, what is the the purpose of the `Serial number` field? I think this is where my brain kind of grinds to a halt.

Anyhow, for now I will simply ignore the serial number field. Cheerio.
